main/webapp/modules/core/langs/translation-en.json

```diff
@@ -680,6 +681,18 @@
     "core-views/use-values-as-identifiers2": "Mark cells as reconciled with their values as identifiers",
     "core-views/choose-reconciliation-service": "Choose a reconciliation service",
     "core-views/choose-reconciliation-service-alert": "Please choose a reconciliation service first.",
+    "core-views/column-join": "Join columns",
+    "core-views/column-join-before-column-picker": "Select and order columns to join",
+    "core-views/column-join-before-options": "Select options",
+    "core-views/column-join-field-separator": "Separator between the content of each column :",
+    "core-views/column-join-field-separator-advice": "Enter one or more character, or keep blank to juxtapose the columns.",
+    "core-views/column-join-replace-nulls": "Replace nulls with...",
+    "core-views/column-join-skip-nulls": "Skip nulls.",
+    "core-views/column-join-replace-nulls-advice": "Enter one or more character, or keep blank to replace nulls with blank strings.",
+    "core-views/column-join-skip-nulls-advice": "Skipping nulls will make impossible the reverse the join.",
+    "core-views/column-join-write-active-column": "Write result in active column.",
```

Comment:
change word choice from "active" to "selected".  This will help with our future UI changes anticipated for multi-column support.

Reply:
ok. Are you refering to an open issue?

Reply:
Yes multi-column operations are captured as an enhancement in multiple issues as well as our project plans.